lib/yoti/activity_details.rb

```diff
@@ -103,7 +103,7 @@ def initialize(receipt, decrypted_profile = nil, decrypted_application_profile =
       @timestamp = receipt['timestamp'] ? Time.parse(receipt['timestamp']) : nil
       @extended_user_profile = process_decrypted_profile(decrypted_profile)
       @extended_application_profile = process_decrypted_profile(decrypted_application_profile)
-      @extra_data = Share::ExtraData.new(extra_data) if extra_data
+      @extra_data = extra_data
```

Comment:
It would be good to add in the missing _param_ docs for this method (looks like `decrypted_application_profile` is missing too)